src/sdam/topology.ts

```diff
@@ -883,7 +877,7 @@ function updateServers(topology: Topology, incomingServerDescription?: ServerDes
   }
 }
 
-function drainWaitQueue(queue: List<ServerSelectionRequest>, err?: MongoDriverError) {
+function drainWaitQueue(queue: List<ServerSelectionRequest>, drainError: MongoDriverError) {
```

Comment:
🔥 I always appreciate drive-by improvement of variable names, thanks for improving this one!

lack of attention to variable names leads to death of readability by a thousand cuts 🔪